### ISSUE TITLE
check min version

### ISSUE THREAD

**Issue by milad1378yz** [huggingface/transformers#17396]
### System Info

```shell
Hi, in transformers/examples/pytorch/text-classification/run_glue.py in line 50, there is a line of code :
'''
check_min_version("4.20.0.dev0")
'''
and there is no 4.20.0 version so I think it should be corrected.
```

### Information

- [ ] The official example scripts
- [ ] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [ ] My own task or dataset (give details below)

### Reproduction

python transformers/examples/pytorch/text-classification/run_glue.py \
  --model_name_or_path bert-base-cased \
  --train_file ./train.csv  \
  --validation_file ./test.csv \
  --do_train \
  --do_eval \
  --evaluation_strategy epoch \
  --max_seq_length 128 \
  --per_device_train_batch_size 32 \
  --learning_rate 5e-5 \
  --max_seq_length 128 \
  --num_train_epochs 2 \
  --seed 2021\
  --output_dir /yazdani/tmp/imdb/

### Expected behavior

```shell
change this part to 
'''
check_min_version("4.18.0.dev0")
'''
```
